internal/domain/application_oidc.go

```diff
@@ -163,6 +164,15 @@ func (a *OIDCApp) OriginsValid() bool {
 	return true
 }
 
+func (a *OIDCApp) AllowedScopePrefixesValid() bool {
+	for _, prefix := range a.AllowedScopePrefixes {
+		if !strings.Contains(prefix, ":") || strings.HasPrefix(prefix, "urn:zitadel:") || strings.HasPrefix("urn:zitadel:", prefix) {
```

Comment:
Not required, but maybe there should be an error related to the validity of the application configuration, to provide information on why the application was not created for example.